wandb/apis/public/api.py

```diff
@@ -1048,3 +1048,34 @@ def list_jobs(self, entity, project):
             return [x["node"]["artifacts"] for x in artifacts]
         except requests.exceptions.HTTPError:
             return False
+
+    @normalize_exceptions
+    def artifact_exists(self, name: str, type: Optional[str] = None):
+        """Return whether an artifact exists.
```

Comment:
can we make this description a bit more specific? Does it check existence in the entity? project? collection? Is it checking whether a version exists, or a collection exists?

Reply:
Checks if an artifact version exists within a specified project and entity. If 'entity' or 'project' is not provided, the function infers these from the default settings. The default for 'project' is "uncategorized" if not specified.